src/client.js

```diff
@@ -138,3 +145,4 @@ module.exports = { SkynetClient, buildRequestHeaders };
 require("./download.js");
 require("./encryption.js");
 require("./upload.js");
+const { nodejs_db_setJSON } = require("./skydb.js");
```

Comment:
Can it just be this?

```suggestion
require("./skydb.js");
```